modules/ref_arch_setup/tasks/install_pe.json

```diff
@@ -0,0 +1,14 @@
+{
+  "description": "Installs PE master on a node",
+
+  "parameters": {
+    "installer_path": {
```

Comment:
Should this param be something like pe_tarball_path based on the description calling it the pe tarball?